src/redux/books/booksSlice.js

```diff
@@ -1,45 +1,32 @@
-import { createSlice } from '@reduxjs/toolkit';
-import { v4 as uuidv4 } from 'uuid';
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
 
-const initialState = [
-  {
-    id: uuidv4(),
-    title: 'The Hunger Games',
-    genres: 'Action',
-    author: 'Suzanne Collins',
-    progres: 64,
-    chapter: 'Chapter 17: "A Lesson Learned"',
-  },
-  {
-    id: uuidv4(),
-    title: 'Dune',
-    genres: 'Fantasy',
-    author: 'Frank Herbert',
-    progres: 8,
-    chapter: 'Chapter 3: "A Lesson Learned"',
-  },
-  {
-    id: uuidv4(),
-    title: 'Capital in the Twenty-First Century',
-    genres: 'Economics',
-    author: 'Suzanne Collins',
-    progres: 0,
-    chapter: 'Introduction: "A Lesson Learned"',
-  },
-];
+const initialState = {
+  Books: [],
+};
```

Comment:
- - Screenshot refernce:
![Screenshot from 2023-09-30 20-27-58](https://github.com/Kingno3l/Bookstore/assets/107632780/f46d1c07-4607-48b3-90f4-1b00c8cd589c)

- [ ] It seems you didnt get the suggestion from the previous reviewer correctly, you need to have the book list here inside your bookSlice, not in the book component. What you are supposed to be doing in the BookList component is reading out the book list using `useSelectore.` Herws what your book slice initial state should look like.
```sh
const initialState = {
  books: [
    {
      item_id: 'item1',
      title: 'The Great Gatsby',
      author: 'John Smith',
      category: 'Fiction',
    },
    {
      item_id: 'item2',
      title: 'Anna Karenina',
      author: 'Leo Tolstoy',
      category: 'Fiction',
    },
    {
      item_id: 'item3',
      title: 'The Selfish Gene',
      author: 'Richard Dawkins',
      category: 'Nonfiction',
    },
  ],
};
```

Then, in your `BookList` component, you can access the book list and then map over them to display them in the UI. Heres what the code snippet should look like. :+1: :+1: 

```sh
import { useSelector } from 'react-redux';
const booksInfo = useSelector((state) => state.books.books);
```

Also as the previous reviewer mentioned, you need to dispatch an event using redux `import { useDispatch } from 'react-redux';` in your books component when the delete button is clicked and pass in the id of the book so that the correct book can be removed. :+1:

Reply:
- [ ] kindly implement the requested change here by the previous reviewer in this comment [here](https://github.com/Kingno3l/Bookstore/pull/5#discussion_r1342001303)